ddp-workspace/projects/ddp-sdk/src/lib/services/authentication/auth0Adapter.service.ts

```diff
@@ -366,11 +366,8 @@ export class Auth0AdapterService implements OnDestroy {
     }
 
     private handleExpiredTemporarySession(): void {
-        this.log.logToCloud(`${this.LOG_SOURCE} expired temporary session`).pipe(take(1)).subscribe(() => {
-                this.session.clear();
-                window.location.reload();
-            }
-        );
+        this.session.clear();
```

Comment:
Looks good, maybe just tighten up the blank lines?

Reply:
🧐 I don't find any blank lines. I think it's the git-diff viewer that gives a false visual effects.

<img width="464" alt="Screenshot 2023-11-28 at 2 50 57 PM" src="https://github.com/broadinstitute/ddp-angular/assets/35533885/8086f04e-fb9a-4caa-9c1e-a532891f3489">